setup.py

```diff
@@ -6,6 +6,9 @@
       author="Ewen Cheslack-Postava",
       author_email='ewen@confluent.io',
       platforms=["any"], 
+      entry_points={
+        'console_scripts': ['ducktape=ducktape.__main__:main'],
```

Comment:
I actually haven't setup a script like this, so I'm not sure, but can we/would it be better to put it under its own module instead of in the `__init__.py` file for the ducktape module? Usually it's not a great idea to put too much code into an `__init__.py` file because they're really intended for configuration of the module and so that code isn't very easily discoverable. I'm not sure how the location interacts with this console_scripts setting, but it looks like we could probably put that code anywhere we want.